source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -421,6 +371,43 @@ This example specifies the following driver behavior in a pool of
    :emphasize-lines: 3-5
    :dedent:
 
+.. _mcs-logger-settings:
+
+Logger Settings
+---------------
+
+Chain the `applyToLoggerSettings() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html#applyToLoggerSettings(com.mongodb.Block)>`__
+method to modify the logging behavior of the driver.
+
+The following table describes all the methods you can chain to your
```

Comment:
Suggestion: you can decide if this is in scope, but I think "all the methods" is awkward. 

```suggestion
The following table describes the methods you can chain to your
```